v63003/outref/gtm8850.txt

```diff
@@ -0,0 +1,40 @@
+# Start background script that does MUPIP FREEZE -ONLINE -ON -NOAUTORELEASE
+# Start 8 background scripts that starts MUMPS processes which do two updates and terminate in a loop
+# Run background scripts for 15 seconds to see if there is a hang
+# Signal background scripts to stop
+# Confirm all background processes finished properly
+
+# Status for freeze process
+# Freeze process finished properly
```

Comment:
A line indicating all feeze and mumps procesess finished properly is good enough. No need to have status for each process 0, 1, 2 etc. below. That is needed only if there is an error.